package/fedora/freecad.spec

```diff
@@ -1,104 +1,69 @@
-
 # Some configuration options for other environments
-# rpmbuild --with=bundled_zipios:  use bundled version of zipios++
-%global bundled_zipios %{?_with_bundled_zipios: 1} %{?!_with_bundled_zipios: 1}
+# rpmbuild --without=bundled_zipios: don't use bundled version of zipios++
+%bcond_without  bundled_zipios
 # rpmbuild --with=bundled_pycxx:  use bundled version of pycxx
-%global bundled_pycxx %{?_with_bundled_pycxx: 1} %{?!_with_bundled_pycxx: 0}
+%bcond_with bundled_pycxx
 # rpmbuild --without=bundled_smesh:  don't use bundled version of Salome's Mesh
-%global bundled_smesh %{?_without_bundled_smesh: 0} %{?!_without_bundled_smesh: 1}
-
-# Prevent RPM from doing its magical 'build' directory for now
-%global __cmake_in_source_build 0
-
-# See FreeCAD-main/src/3rdParty/salomesmesh/CMakeLists.txt to find this out.
-%global bundled_smesh_version 7.7.1.0
+%bcond_without bundled_smesh
+# rpmbuild --without=tests:  esclude tests in %%check
+%bcond_without tests
+# rpmbuild --without=bundled_gtest:  don't use bundled version of gtest and gmock
+%bcond_without bundled_gtest
+# rpmbuild --without=bundled_gtest:  don't build debug information
+%bcond_without debug_info
 
-# Some plugins go in the Mod folder instead of lib. Deal with those here:
-%global mod_plugins Mod/PartDesign
-%define name freecad
-%define github_name FreeCAD
-%define branch main
 
-Name:           %{name}
+Name:           freecad
 Epoch:          1
-Version:        1.1.0
-Release:        pre_{{{git_commit_no}}}%{?dist}
+Version:        1.1.0~dev
+Release:        %autorelease
+
 Summary:        A general purpose 3D CAD modeler
 Group:          Applications/Engineering
-
-License:        LGPLv2+
+License:        GPL-2.0-or-later
 URL:            https://www.freecad.org/
-Source0:        https://github.com/%{github_name}/FreeCAD/archive/%{branch}.tar.gz
 
+Source0:        https://github.com/FreeCAD/FreeCAD-Bundle/releases/download/weekly-builds/freecad_source.tar.gz
 
-# Utilities
-BuildRequires:  cmake gcc-c++ gettext
-BuildRequires:  doxygen swig graphviz
-BuildRequires:  gcc-gfortran
-BuildRequires:  desktop-file-utils
-BuildRequires:  git
 
-BuildRequires:  tbb-devel
+# Maintainers:  keep this list of plugins up to date
+# List plugins in %%{_libdir}/%%{name}/lib, less '.so' and 'Gui.so', here
+%global plugins AssemblyApp AssemblyGui CAMSimulator DraftUtils Fem FreeCAD Import Inspection MatGui Materials Measure Mesh MeshPart Part PartDesignGui Path PathApp PathSimulator Points QtUnitGui ReverseEngineering Robot Sketcher Spreadsheet Start Surface TechDraw Web _PartDesign area flatmesh libDriver libDriverDAT libDriverSTL libDriverUNV libE57Format libMEFISTO2 libSMDS libSMESH libSMESHDS libStdMeshers libarea-native
```

Comment:
The description above makes it sound like there shouldn't be the "Gui" entries here. Also, why does PartDesign appear out of order, and with an underscore?

Reply:
it is the library name to keep the library private and avoid other package to depend on that library. 
The gui is added in the regular expression used.
This mechanism is from the original spec file.
I'm not sure it is needed.

Reply:
This list is used to filter the Requires and provides with a regular expression that add ".so" and "Gui.so" to  names in the list. Removing this libraries from Provides prevent other packages to depend on that libraries (If a package try to use a library that are filtered it can't be installed because no package declared that provides the library).
The package with Gui don't have a library counterpart without trailing Gui  
_PartDesign.so is the library generated by the build. 
This list didn't change from original spec file. 
Let me know if i need to fix the list removing the trailing Gui and sort it. Or remove this part completely and use the default behaviour of rpmbuild to declare the libraries as provided.

Reply:
If this is just leftover from the old version of the file I don't mind leaving it, but of course feel free to clean it up if you'd like.